docs/overview.md

```diff
@@ -56,7 +56,7 @@ batteries, which are enumerated below.
   on-chain logic - without needing to move funds in and out of a separate
   lightning wallet with on-chain transactions and a separate private key system.
   * LDK does not currently provide a sample wallet module, but its sample node
-    implementation uses Bitcoin Core's wallet for UTXO management e.g. [here](https://github.com/TheBlueMatt/rust-lightning-bitcoinrpc/blob/139a653eeba313284c6d9d2eb2776d30dbb0ca3d/src/main.rs#L219)
```

Comment:
Doesn't matter too much but instead of "e.g." we could say "here and elsewhere" for consistency w/ the rest of the doc